templates/data/generic_module_data.yaml

```diff
@@ -1,12 +1,9 @@
 templates:
   - from: generic.tmpl
     render:
-      - to: ../../prow/jobs/modules/external/keda-manager/keda-manager-generic.yaml
+      - to: ../../prow/jobs/modules/external/keda-manager.yaml
         localSets:
-          jobConfig_default:
-            imagePullPolicy: "Always"
-            privileged: "false"
```

Comment:
Here you remove localSet with privileged property set to false.

Reply:
I removed it because this localSet was not used (it was an orphan). But you are right. Maybe it's a good idea to keep such a configuration in the specified localSet. I will change it 💪